setup.lua

```diff
@@ -43,8 +43,8 @@ local successful = pcall(function()
     print("Extracting")
     shell.execute("tar -xf NIDAS.tar")
     filesystem.remove(workDir .. "NIDAS.tar")
-    filesystem.remove("/home/lib")
-    shell.execute("cp -r lib /home/lib")
+    --filesystem.remove("/home/lib")
+    --shell.execute("cp -r lib /home/lib")
```

Comment:
It makes sure you can import packages from lib without using `lib.` in your requires